crawler/plugins/applications/util.py

```diff
@@ -0,0 +1,63 @@
+import dockerutils
+
+
+class Inspector(object):
+    '''
+    Interface class.
+    '''
+
+    def __init__(self, container_id):
+        self.inspect = dockerutils.exec_dockerinspect(container_id)
+        # self.cotainers = dockerutils.exec_dockerlistcontainers()
```

Comment:
OK, I will remove it.

I prefer to add `get_ip` and `get_ports` methods in DockerContainer class. `is_app_container` method is not needed to add actually. Instead, I plan to use `dockercontainer.image` variable.  

So can I include these two methods? And then, I prefer to use `dockercontainer class` or `dockerutils.py` instead of `app/util.py`.